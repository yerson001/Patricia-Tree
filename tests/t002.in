I 4985
I 8340
I 8142
I 8424
I 7375
I 9103
I 8261
I 8123
I 929
I 5498
I 8659
I 2392