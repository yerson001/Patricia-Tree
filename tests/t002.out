I 4985
Successful insertion of 4985 with node bit test 12
I 8340
Successful insertion of 8340 with node bit test 13
I 8142
Successful insertion of 8142 with node bit test 11
I 8424
Successful insertion of 8424 with node bit test 6
I 7375
Successful insertion of 7375 with node bit test 9
I 9103
Successful insertion of 9103 with node bit test 9
I 8261
Successful insertion of 8261 with node bit test 7
I 8123
Successful insertion of 8123 with node bit test 6
I 929
Successful insertion of 929 with node bit test 9
I 5498
Successful insertion of 5498 with node bit test 10
I 8659
Successful insertion of 8659 with node bit test 8
I 2392
Successful insertion of 2392 with node bit test 11
